src/components/EmpLogin/EmpLogin.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FormEvent, useState } from 'react';
+import img from '../../images/bg-image.jpg';
+import { useNavigate } from 'react-router-dom';
+import Footer from '../Footer';
+import './EmpLogin.css'
+import { apiRequest } from '../../HelperFunction/helperFunction';
+
+type User = {
+    userName: string,
+    password: string
+}
+
+const EmpLogin = () => {
+    const Navigate = useNavigate();
+
+    const [details, setDetails] = useState<User>({
+        userName: '',
+        password: '',
+    });
+
+    const handleDataChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setDetails({ ...details, [e.target.name]: e.target.value })
+    }
+
+    async function handleLogin(event: FormEvent<HTMLFormElement>) {
+        event.preventDefault();
+        try {
+            const res = await apiRequest(`${process.env.REACT_APP_URL}/employee/empLogin`, 'post', details);
+            localStorage.setItem('token', res.data.token);
```

Comment:
error handling needs to be done properly.
if the res is undefined, what may happen?